repositories.yaml

```diff
@@ -137,23 +137,23 @@ firefox-reality:
     - org.mozilla.components:service-glean
     - org.mozilla.components:support-sync-telemetry
 fogotype:
-  app_id: org.mozilla.fogotype
```

Comment:
fly-by: can't the probe-scraper do the transformation itself? It might be clearer to specify the same app ids that the app advertised in this repository file...

Reply:
Yeah, that is how we should handle these. However let's first get this moving again then we can move this responsibility to MSG.

Reply:
https://github.com/mozilla/probe-scraper/issues/156